scripts/dts/extract/pinctrl.py

```diff
@@ -5,10 +5,23 @@
 #
 
 from extract.globals import *
+from extract.edts import *
 from extract.directive import DTDirective
 
 ##
-# @brief Manage pinctrl-x directive.
+# @brief Manage pinctrl related directives.
+#
+# Handles:
+# - pinctrl-x
+# - pinctrl-names
+#
+# Generates in EDTS:
+# - pinctrl/<index>/name           : name of the pinctrl
```

Comment:
Since pinctrl api, driver and dts nodes are not in zephyr code , please remove this part.

Reply:
If I would remove the node all pinmux defines would go away also. The current pinmux drivers need these defines. Pinmux API/ drivers are part of Zephyr code.